Plots/Overlays/NCL_overlay_6.py

```diff
@@ -0,0 +1,151 @@
+import matplotlib.colors as mcolors
+from mpl_toolkits.axes_grid1.inset_locator import inset_axes
+import cartopy.crs as ccrs
+import cartopy.feature as cfeature
+
+import geocat.datafiles as gdf
+from geocat.viz import cmaps as gvcmaps
+
+###############################################################################
+# Read in data:
+
+# Open a netCDF data file using xarray default engine and load the data into xarrays
+uf = xr.open_dataset(gdf.get("netcdf_files/Ustorm.cdf"))
+vf = xr.open_dataset(gdf.get("netcdf_files/Vstorm.cdf"))
+pf = xr.open_dataset(gdf.get("netcdf_files/Pstorm.cdf"))
+tf = xr.open_dataset(gdf.get("netcdf_files/Tstorm.cdf"))
+u500f = xr.open_dataset(gdf.get("netcdf_files/U500storm.cdf"))
+v500f = xr.open_dataset(gdf.get("netcdf_files/V500storm.cdf"))
+
+p = pf.p.isel(timestep=0).drop('timestep')
+t = tf.t.isel(timestep=0).drop('timestep')
+u = uf.u.isel(timestep=0).drop('timestep')
+v = vf.v.isel(timestep=0).drop('timestep')
+u500 = u500f.u.isel(timestep=0).drop('timestep')
+v500 = v500f.v.isel(timestep=0).drop('timestep')
+time = vf.timestep
+
+# Convert Pa to hPa
+p = p/100
+# Convert K to F
+t = (t - 273.15) * 9/5 + 32
+
+###############################################################################
+# Create map:
+fig = plt.figure(figsize=(8, 7))
+proj = ccrs.LambertAzimuthalEqualArea(central_longitude=-100,
+                                      central_latitude=40)
+
+# Set axis projection
+ax = plt.axes([0, 0.2, 0.8, 0.7], projection=proj)
+# Create inset axes for color bars
+cax1 = inset_axes(ax, width='5%', height='100%', loc='lower right',
+                  bbox_to_anchor=(0.125, 0, 1, 1),
+                  bbox_transform=ax.transAxes,
+                  borderpad=0)
+
+cax2 = inset_axes(ax, width='100%', height='7%', loc='lower left',
+                  bbox_to_anchor=(0, -0.15, 1, 1),
+                  bbox_transform=ax.transAxes,
+                  borderpad=0)
+
+# Set extent to include roughly the United States
+ax.set_extent((-128, -58, 18, 65), crs=ccrs.PlateCarree())
+
+# Add map features
+transparent = (0, 0, 0, 0)  # RGBA value for a transparent color for lakes
+ax.add_feature(cfeature.OCEAN, color='lightskyblue', zorder=0)
+ax.add_feature(cfeature.LAND, color='silver', zorder=1)
+ax.add_feature(cfeature.LAKES, linewidth=0.5, edgecolor='black',
+               facecolor=transparent, zorder=3)
+ax.add_feature(cfeature.COASTLINE, linewidth=0.5, zorder=3)
+
+#
+# Plot pressure level contour
+#
+p_cmap = gvcmaps.StepSeq25
+pressure = p.plot.contourf(ax=ax, transform=ccrs.PlateCarree(), cmap=p_cmap,
+                           levels=np.arange(975, 1050, 5), add_colorbar=False,
+                           add_labels=False, zorder=2)
+plt.colorbar(pressure, cax=cax1, ticks=np.arange(980, 1045, 5))
+# Format colorbar label
+cax1.yaxis.set_label_text(label='\n'.join('Sea Level Pressure'), fontsize=14,
+                          rotation=0)
+cax1.yaxis.set_label_coords(-0.5, 0.9)
+
```

Comment:
Overall, looks great! I noticed in the original NCL plot the color bars do not have any ticks, if you want to also do this on your plot, I found using/adding cax1.tick_params(size=0) here (line 110) and cax2.tick_params(size=0) after line 140 would remove the tick marks. Up to you if you want to make this change or not!